src/braket/circuits/circuit.py

```diff
@@ -773,6 +837,17 @@ def apply_initialization_noise(
 
         return apply_noise_to_moments(self, noise, target_qubits, "initialization")
 
+    def set_parameter_value(self, param_values: Dict[FreeParameter, Number]):
```

Comment:
No way to unset parameters right now? What if we want to run a parametrized circuit multiple times with different parameter values?

Reply:
Can be called in a loop with parameter values used for execution and sending the circuits with these vals

Reply:
The free parameter value would be overridden with the next value with subsequent set_parameter_value calls